conduct.md

```diff
@@ -13,7 +13,29 @@
 * Communication
 * Participation
 * Decisions
-* Postmortems
+
+## Postmortems
+
+Postmortems are great ways of looking at and analyzing the performance of the
+project after its launch. Through postmortems we can look at what was successful
+and what failed and needs improvement. These discussions also provide great
+insight in how to improve in future projects.
+
+* **Acceptable Conduct (A):**
+  * *Members shall*
+    * Create an agenda of things that need to be reviewed and fixed
+    * Participate in the discussions in a respectful and courteous manner
+    * Provide constructive criticism and propose solutions for issues faced
+    * Receive feedback and work on improving in the future, criticism
+    is not personal
```

Comment:
I would maybe clarify or reword this statement. The second half does not relate to receiving feedback.

Reply:
I tried to clarify these lines. let me know it this is better. Thanks!